src/appleseed/renderer/kernel/rendering/masterrenderer.cpp

```diff
@@ -461,13 +462,24 @@ struct MasterRenderer::Impl
         // Updating the trace context causes ray tracing acceleration structures to be updated or rebuilt.
         m_project.update_trace_context();
 
+        // Load the checkpoint if any.
+        Frame& frame = *m_project.get_frame();
+
```

Comment:
Remove this blank line and the one below so that the comment applies to the whole block.